transforms/images/discard-border-objects-plugin/src/main.py

```diff
@@ -0,0 +1,113 @@
+import argparse, logging, os, time, filepattern
+from pathlib import Path
+import tensorflow as tf
```

Comment:
You don't use tensorflow in this plugin.

Reply:
Done